### PR TITLE
Azure pipelines only build

### LINKED ISSUE #36: Azure Pipelines only build
As a < type of user >, I want < some goal > so that < some reason >.
Fuking Scrum

I as a product_owner/analyst/developer/tester/user, I want to do Azure Pipelines only build so that i can do Azure pipelines only build

----

OK but now serious

I want to implement Continous Integration on Azure pipelines as part of the epic #35

In order to achieve this task I will copy and paste this pipeline from the Azure documentation:
https://github.com/MicrosoftDocs/pipelines-android/blob/master/azure-pipelines.yml
and read this:
https://docs.microsoft.com/en-us/azure/devops/pipelines/ecosystems/android?view=azure-devops

The difficulty of this task is 0.

### PR DESCRIPTION
**Pull request by mercuriete**
* Closes #36
* Copy azure pipeline from https://github.com/MicrosoftDocs/pipelines-android/blob/master/azure-pipelines.yml
* Fix vmImage to macos-latest
* drop tests and drop junit xml publishing

### CODE REVIEW

File: azure-pipelines.yml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+          jdkVersionOption: '1.8'
+          jdkArchitectureOption: 'x64'
+          publishJUnitResults: true
+          testResultsFiles: '**/TEST-*.xml'
+          tasks: 'assembleDebug test'
+      displayName: gradlew assembleDebug test
+
+    - task: CopyFiles@2
+      inputs:
+          contents: '**/*.apk'
```
Comment (alu0100207385) [inline azure-pipelines.yml:31]: consider updating this to bundle in the future

Comment (mercuriete) [reply azure-pipelines.yml:31]: good catch.
I am using bundle.
It is a copy paste error LOL!!!!!

Comment (mercuriete) [reply azure-pipelines.yml:31]: We are already generating apk as well.
I will leave this task for the next step on this epic.
This task is only to run and configure azure pipelines as fast as I can.

I will close this conversation.

Comment (alu0100207385) [reply azure-pipelines.yml:29]: the horse way!

**Pull request merged**